python/chronos/src/bigdl/chronos/metric/forecast_metrics.py

```diff
@@ -259,3 +267,89 @@ def get_latency(func, *args, num_running=100, **kwargs):
                         "p99": round(1000 * sorted_time[int(0.99 * num_running)], 3)}
 
         return latency_list
+
+    @staticmethod
+    def plot(y,
+             std=None,
+             ground_truth=None,
+             x=None,
+             feature_index=0,
+             layout=(4,2),
+             prediction_interval=0.95,
+             figsize=(16,16),
+             output_file=None,
+             **kwargs):
+        '''
+        `Evalutor.plot` function helps users to visualize their forecasting result.
+
+        :param y: predict result, a 3-dim numpy ndarray with shape represented as
+               (batch_size, predict_length, feature_dim).
+        :param std: standard deviation, a 3-dim numpy ndarray with shape represented
+               as (batch_size, predict_length, feature_dim). Same shape as `y`.
+        :param ground_truth: ground truth, a 3-dim numpy ndarray with shape represented as
+               (batch_size, predict_length, feature_dim). Same shape as `y`.
+        :param x: input numpy array, a 3-dim numpy ndarray with shape represented
+               as (batch_size, lookback_length, feature_dim).
+        :param feature_index: a tuple/int, the instance index (along first dim) to plot.
+               Default to the first index.
+        :param layout: a 2-dim tuple, indicate the row_num and col_num to plot. Default to
+               automatically row_num and col_num. 
+        :param prediction_internval: a float, indicates the confidence percentile. Default to
+               0.95 refer to 95% confidence. This only effective when `std` is not None.
+        :param figsize: figure size to be inputed to pyplot. Default to (16,16).
+        :param output_file: a path, indeicates the save path of the output plot. Default to
+               None, indicates no output file is needed.
+        :param **kwargs: other paramters will be passed to matplotlib.pyplot.
+        '''
+        import matplotlib.pyplot as plt
+
+        # shape check
+        if std is not None:
+            _check_shape(y, std, "y", "std")
+        if ground_truth is not None:
+            _check_shape(y, ground_truth, "y", "ground_truth")
+        
+        batch_num = y.shape[0]
+        horizon = y.shape[1]
+        lookback = 0 if x is None else x.shape[1]
+        row_num = 1 if layout is None else layout[0]
+        col_num = 1 if layout is None else layout[1]
+
+        y_index = list(range(lookback, horizon+lookback))
+        x_index = list(range(0, lookback))
+        iter_num = 1
+
+        plt.figure(figsize=figsize, **kwargs)
+
+        for row_iter in range(1, row_num+1):
+            for col_iter in range(1, col_num+1):
+                instance_index = random.randint(0, y.shape[0])
```

Comment:
I think 'instance_index' may be customized by users

Reply:
yeah, I remove it in the last minute since I don't want to check if the index's length fits the layout :( , I will add it.